### PR TITLE
fix various typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -S *.ts,thirdparty, -L appy,ba,inbetween,inout,pevent,possibile,upto`

### CODE REVIEW

File: toonz/sources/stdfx/igs_line_blur.cpp [hunk 1]
```diff
@@ -3049,7 +3049,7 @@ void pixel_line_root::exec09_same_way_expand(
     }
   }
 
-  /* 各端点(one,anohter)ごと
+  /* 各端点(one,ananother)ごと
```
Comment (RodneyBaker) [inline toonz/sources/stdfx/igs_line_blur.cpp:3052]: This appears to be a typo fix that still leaves the word typo'd but in another form.
As a comment not a high priority but should just be 'another'.

File: toonz/sources/stdfx/igs_line_blur.cpp [hunk 2]
```diff
@@ -3026,7 +3026,7 @@ void pixel_line_root::exec09_same_way_expand(
                        clp_select->get_i32_count_max());
   }
 
-  /* 各端点(one,anohter)ごと
+  /* 各端点(one,another)ごと
```
Comment (RodneyBaker) [inline toonz/sources/stdfx/igs_line_blur.cpp:3029]: Another automated typo fix that leaves the word typo'd in yet another form.
See also line 3052.

File: toonz/sources/include/tsound_t.h [hunk 3]
```diff
@@ -85,9 +85,9 @@ class DVAPI TSoundTrackT final : public TSoundTrack {
   //----------------------------------------------------------------------------
 
   /*!
-Returns a soundtrack whom is a clone of the object for the spicified channel
-A clone means that it's an object who lives indipendently from the other
-from which it's created.It hasn't reference to the object.
+Returns a soundtrack whom is a clone of the object for the spicified channel.
```
Comment (RodneyBaker) [inline toonz/sources/include/tsound_t.h:88]: The automated typo fix here failed at the word 'spicified' but seems to have fixed the remainder of the paragraph.

**Comment by RodneyBaker** [comment]
Very nice @luzpaz 
I only found the two places where the typos were replaced but with words that themselves are typos.

It appears the appveyor build might have failed simply due to the number of files having a change.
WIll wait to see if the Github Actions build passes.

Thanks much for pressing forward with codespell.  :)

File: toonz/sources/include/tsound_t.h (hunk 3 above)
Comment (luzpaz) [reply toonz/sources/include/tsound_t.h]: Will update the codespell dict to add `spicified`! Thanks. Fixed in 69208c7

File: toonz/sources/stdfx/igs_line_blur.cpp (hunk 1 above)
Comment (luzpaz) [reply toonz/sources/stdfx/igs_line_blur.cpp]: Apologies. Fixed in 69208c7

**Comment by luzpaz** [comment]
I have some source code typos to submit. Should I open a separate PR for them?

**Comment by RodneyBaker** [comment]
> I have some source code typos (likely to break API) to submit. Should I open a separate PR for them?

Yes, please.  That isn't strictly necessary but in that way we can go ahead and merge this PR.

**Comment by RodneyBaker** [comment]
LGTM
Thank you @luzpaz for the fixed typos!

**Comment by beeheemooth** [comment]
Hopefully the program will understand what the developers mean in the comments and will work better!  :)

**Pull request merged**